File: MCP_Backend-master/src/main/java/com/greybox/mediums/models/equiweb/CIStatementResponse.java
package com.greybox.mediums.models.equiweb;

import lombok.Data;

import java.math.BigDecimal;

@Data
public class CIStatementResponse {
    private String transDate;
    private String effectiveDate;
    private String reference;
    private String credit;
    private String debit;
    private BigDecimal txnAmount;
    private BigDecimal closing;
    private String description;
}

File: micropay-mobile-apis-master/Models/CIStatementResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace micropay_apis.Models
{ 
	public class CIStatement
	{
		public string transDate;
		public string effectiveDate;
		public string reference;
		public decimal? credit;
		public decimal? debit;
		public decimal? txnAmount;
		public decimal? closing;
		public string description;
	}
	public class CIStatementResponse
	{
		public ResponseMessage response;
		public List<CIStatement> data; 
	}
}